src/services/qix/index.js

```diff
@@ -90,7 +96,7 @@ export default class Qix {
   */
   buildUrl(sessionConfig, appId) {
     const { secure, host, port, prefix, subpath, route, identity,
-      reloadURI, urlParams } = sessionConfig;
+      reloadURI, urlParams, ttl } = sessionConfig;
```

Comment:
Should the **ttl** be defined in the @typedef Configuration
https://github.com/qlik-oss/enigma.js/blob/270e3d65ceeb047e19cbee7190805d9bc938ec36/src/services/qix/index.js#L14